lib/src/game_storage.dart

```diff
@@ -50,27 +50,28 @@ class GameStorage {
     targetPlatform.clearValues();
   }
 
-  int _getIntValue(String key, [int defaultValue = 0]) {
-    assert(key != null);
-
+  int? _getIntValue(String key, [int? defaultValue = 0]) {
     final strValue = targetPlatform.getValue(key);
     return _parseValue(strValue, defaultValue);
   }
 
   void _setIntValue(String key, int value) {
     assert(key != null);
```

Comment:
@scheglov – same here. Not sure if analyzer warns on `assert(true)` but it should – and this, too!